tests/describe_task_queue.go

```diff
@@ -38,28 +38,62 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 )
 
+// making a BaseSuite
 type (
-	DescribeTaskQueueSuite struct {
```

Comment:
The re-structuring of this file, to go with this base struct which gets inherited by the other suites, was done due to the following:

- Initially, I wanted to change the dynamic config pertaining to the cache TTL to test out cache evictions. However, I soon realized that overriding the dynamic config in `setupTest` simply doesn't work since the cache is initialized during `suiteSetup`. 
- Thus, adding a new suite (DescribeTaskQueueSuiteWithCache) was the idea that I had. However, instead of making two new files and copying code from here, I went ahead with re-structuring this file by using some basic inheritance properties.

cc - @ShahabT since you had written these tests (although I have changed no content in the pre-existing ones)